gqcp/include/QCMethod/HF/UHF/UHFSCFEnvironment.hpp

```diff
@@ -99,7 +99,7 @@ class UHFSCFEnvironment {
     UHFSCFEnvironment(const QCModel::RHF<Scalar>& rhf_parameters, const USQHamiltonian<Scalar>& sq_hamiltonian, const ScalarUSQOneElectronOperator<Scalar>& S) :
         UHFSCFEnvironment(rhf_parameters.numberOfElectrons(Spin::alpha), rhf_parameters.numberOfElectrons(Spin::beta),
                           sq_hamiltonian, S,
-                          rhf_parameters.expansion().matrix(), rhf_parameters.expansion().matrix()) {}
+                          UTransformation<Scalar> {rhf_parameters.expansion().matrix(), rhf_parameters.expansion().matrix()}) {}
```

Comment:
There's a short-hand named constructor for this:
```suggestion
                          UTransformation<Scalar>::FromEqual(rhf_parameters.expansion().matrix())) {}
```